### PR TITLE
feat: Review endpoints

### PR DESCRIPTION
**Pull request by duolok**
This PR adds endpoints for all operations with reviews.

### CODE REVIEW

File: src/main/java/com/lunark/lunark/model/ReviewReport.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package com.lunark.lunark.model;
+
+import java.time.LocalDateTime;
+
+public class ReviewReport {
+    private Long id;
+    private LocalDateTime date;
+
```
Comment (MomirMilutinovic) [inline src/main/java/com/lunark/lunark/model/ReviewReport.java:8]: Shouldn't ReviewReport reference Review?
```suggestion
private Review review;
```

File: src/main/java/com/lunark/lunark/controller/ReviewController.java
Comment (MomirMilutinovic) [inline src/main/java/com/lunark/lunark/controller/ReviewController.java:1]: I think we also need endpoints for:
- Getting all reviews for a given host (/api/reviews/host/{host_id})
- Getting all reviews for a given property (/api/reviews/property/{property_id})
- Reporting a review (/api/reviews/report/{id})
- Approving a review (/api/reviews/approve/{id})

**Review by MomirMilutinovic — APPROVED** [review]
Great work. I would like to suggest a few additions. See the comments below.

File: src/main/java/com/lunark/lunark/model/ReviewReport.java (hunk 1 above)
Comment (duolok) [reply src/main/java/com/lunark/lunark/model/ReviewReport.java:8]: You're right. I will refactor the code soon.

**Pull request merged**